tests/settings.py

```diff
@@ -1,17 +1,24 @@
 # -*- coding: utf-8 -*-
 import os
 import sys
-
+import django
 
 urlpatterns = []
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': ':memory:'
+if django.VERSION[:2] >= (3, 0):
```

Comment:
why is this needed?

Reply:
https://docs.djangoproject.com/en/3.0/releases/3.0/#features-removed-in-3-0
django.db.backends part is removed

Reply:
I only see `The django.db.backends.postgresql_psycopg2 module is removed.`. they removed  in favor of `postgresql`